platform/util/platform-util/src/main/java/org/codice/ddf/platform/util/RandomNumberGenerator.java

```diff
@@ -22,11 +22,11 @@
 
 public class RandomNumberGenerator {
```

Comment:
It was just added in another PR a few days ago. We also only do that if a public API has been released.